R/fda-table_8.R

```diff
@@ -0,0 +1,56 @@
+# Pre-processing
+library(dplyr)
+library(scda)
+library(tern)
+library(rlistings)
+
+# Deaths
+adae <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adae") %>%
+  filter(SAFFL == "Y" & AESDTH == "Y") %>%
+  select(
+    STUDYID, USUBJID, ACTARM, ACTARMCD, AGE, SEX, AETERM, AEDECOD, ASTDY, AESDTH, AEOUT, DTHADY
+  ) %>%
+  mutate(AGESEX = paste0(AGE, "/", SEX)) %>%
+  distinct(USUBJID, AETERM, AEDECOD, DTHADY, .keep_all = TRUE)
+
+# Dosing
+adex <-
+  synthetic_cdisc_dataset("rcd_2022_06_27", "adex") %>%
+  filter(PARAMCD == "TDOSE") %>%
+  select(USUBJID, PARAMCD, PARAM, AVAL, AVALU, TRTSDT, TRTEDT) %>%
+  mutate(
+    DOSDUR = (TRTEDT - TRTSDT + 1),
```

Comment:
Again, convert numeric variable to character to handle any missing values
```suggestion
    DOSDUR = (TRTEDT - TRTSDT + 1) %>% as.character(),
```